packages/fuels-signers/src/wallet.rs

```diff
@@ -288,13 +290,58 @@ impl Wallet {
         ];
 
         // Build transaction and sign it
-        let mut tx =
-            self.get_provider()
-                .unwrap()
-                .build_transfer_tx(&inputs, &outputs, tx_parameters);
-        let _sig = self.sign_transaction(&mut tx).await.unwrap();
+        let mut tx = self
+            .get_provider()?
+            .build_transfer_tx(&inputs, &outputs, tx_parameters);
+        let _sig = self.sign_transaction(&mut tx).await?;
 
-        let receipts = self.get_provider().unwrap().send_transaction(&tx).await?;
+        let receipts = self.get_provider()?.send_transaction(&tx).await?;
+
+        Ok((tx.id().to_string(), receipts))
+    }
+
+    /// UNCONDITIONAL transfer of `amount` coins of type `asset_id` to
+    /// the contract at `to`.
+    /// Fails if amount for asset ID is larger than this wallets spendable coins.
+    /// Returns the transaction ID that was sent and the list of receipts.
+    ///
+    /// CAUTION !!!
+    ///
+    /// This will transfer coins to a contract even with no way to retrieve them
+    /// (i.e. no withdrawal functionality on receiving contract), possibly leading
+    /// to the PERMANENT LOSS OF COINS if not used with care.
+    pub async fn force_transfer_to_contract(
+        &self,
+        to: &ContractId,
+        amount: u64,
```

Comment:
Is `amount` really the number of coins? It sounds like it's balance, and not the number of coins (ie UTXOs). The wording should be improved around this.

Reply:
Depends IIUC, from the perspective of the wallet, `amount` will be deducted, from the contracts perspective, `balance` will be added. Since this is a `wallet` method i think its fine to keep it as `amount`

Reply:
What I mean is `amount` coins means the total balance change is going to be: 
Sum over the coins (UTXOs) of the amount of each coin.
The wording is confusing: it's not the number of UTXOs that matters, but the resulting balance. Each coin has an amount, so sending `amount` of coins, you could send 10 coins of 100 amount each, or 10 coins of amount 1 each.
-> what matters is the balance, not the number (`amount`)of UTXOs.